authors/apps/authentication/urls.py

```diff
@@ -4,9 +4,8 @@
     LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
 )
 
-
 urlpatterns = [
     path('user/', UserRetrieveUpdateAPIView.as_view()),
```

Comment:
The code looks okay except for the `UserRetrieveUpdateAPIView` route which does not have a name parameter, thus running `reverse()` on this route will cause this route to break.